internal/olm/operator/registry/fbcindex/fbc_registry_pod.go

```diff
@@ -115,6 +119,15 @@ func (f *FBCRegistryPod) Create(ctx context.Context, cfg *operator.Configuration
 		return nil, fmt.Errorf("error setting owner reference: %w", err)
 	}
 
+	// Add security context if the user passed in the --security-context-config flag
+	if f.SecurityContext == "restricted" {
```

Comment:
Nit - Since we define a constant for this in the `registry` package should we use the constant for comparison instead of an explicit string? Maybe:
```suggestion
	if f.SecurityContext == registry.Restricted {
```

Reply:
@everettraven there's a circular dependency. That's why I did'nt use the constant. And I was trying hard not to refactor the code :)

Reply:
@jmrodri Ahh, gotcha. Then it is good to go :)